spec/system/page_objects/unhandled_tagger.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class UnhandledTagger < PageObjects::Pages::Base
+      def handle
+        page.find(".topic-footer-main-buttons .handle").click
```

Comment:
Maybe `.topic-footer-main-buttons` could be abstracted here?

The only thing we are checking is `.handle` and `.unhandle`

code-golf, too, but why not have these `click` methods called `click_handle` and `click_unhandle`?

Reply:
mostly because that wouldn't make my spec shorter atm, I have no need to call them directly for now